CHANGELOG.md

```diff
@@ -71,6 +71,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Workspace] Add create workspace page ([#6179](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6179))
 - [Multiple Datasource] Make sure customer always have a default datasource ([#6237](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6237))
 - [Workspace] Add workspace list page ([#6182](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6182))
+- [Workspace] Add duplicate saved objects API ([#6288](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6288))
```

Comment:
This is little bit confusing to me. How about "Add API to copy saved objects between workspaces"? Also why we are using duplicate keyword and not copy? any specific reason?

Reply:
Because all of the UI mockups use `duplicate`, so I use this wording to make it consistent. I've modified the change log, thank you!